packages/template-retail-react-app/app/pages/account/wishlist/index.jsx

```diff
@@ -16,7 +16,7 @@ import {useWishList} from '@salesforce/retail-react-app/app/hooks/use-wish-list'
 
 import PageActionPlaceHolder from '@salesforce/retail-react-app/app/components/page-action-placeholder'
 import {HeartIcon} from '@salesforce/retail-react-app/app/components/icons'
-import ProductItem from '@salesforce/retail-react-app/app/components/product-item/index'
+import ProductItem from '@salesforce/retail-react-app/app/components/product-item'
```

Comment:
TIL! you can't have index in your import path, is there a bug with the webpack extensibility plugin?

Reply:
I believe it is yes, ticket incoming